### PR TITLE
New file structure

### PR DESCRIPTION
**Pull request by devraulu**
This is the new file structure we agreed on. It is module-based (proposed by Tunde).

### CODE REVIEW

**Comment by ayoola-solomon** [comment]
@devCayacoa could you please check why the builds are failing

File: package.json [hunk 1]
```diff
@@ -1,74 +1,51 @@
...
+			">0.2%",
+			"not dead",
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
+	}
```
Comment (ayoola-solomon) [inline package.json]: Please use 2 line spacing and use space instead of tabs. Checkout [editorconfig](https://editorconfig.org/) for more on how to make sure the same config is being used with the rest of the team

File: src/index.css [hunk 2]
```diff
@@ -1,21 +0,0 @@
-* {
```
Comment (ayoola-solomon) [inline src/index.css]: why did you delete this file?

**Comment by devraulu** [comment]
> @devCayacoa could you please check why the builds are failing

The tests are failing because they expect the GitHub auth to work, something we have yet to implement.
image

File: src/index.css (hunk 2 above)
Comment (devraulu) [reply src/index.css]: I left it empty therefore git didn't add it. I added it back a few minutes ago.

File: package.json (hunk 1 above)
Comment (devraulu) [reply package.json]: Do you have an example of how it is supposed to look like?

Comment (toystars) [reply package.json]: @devCayacoa the file would be named `.editorconfig` and would look like this:

```code
# EditorConfig is awesome: http://EditorConfig.org

# top-most EditorConfig file
root = true

# Unix-style newlines with a newline ending every file
[*]
charset = utf-8
indent_style = space
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true


[**.html]
indent_style = space
indent_size = 4
wrap_line_length = 80
wrap_attributes = force

```

**Comment by toystars** [comment]
> > @devCayacoa could you please check why the builds are failing
> 
> The tests are failing because they expect the GitHub auth to work, something we have yet to implement.
> image

@devCayacoa  Disable the failing test then, since the feature won't be part of this PR

File: package.json (hunk 1 above)
Comment (devraulu) [reply package.json]: I meant how the code should look like. Also, should I apply this style to all of the files?

Comment (toystars) [reply package.json]: The .editorconfig file takes care of the format for you. And yes, it will be applied to all the files

**Review by toystars — APPROVED** [review]
